### PR TITLE
update submission.md

### PR DESCRIPTION
**Pull request by Anandhakrishnan27**
The **Resume API project** is a serverless application built using the **AWS Cloud Development Kit (CDK)**. It deploys a RESTful API that allows users to retrieve resume data by leveraging AWS Lambda for serverless computing and Amazon API Gateway for API management.

Services Used

**AWS Lambda**: Serverless function to retrieve resume data.
**Amazon API Gateway**: RESTful API endpoint for accessing resume data.
**AWS CDK**: Define and provision AWS resources using infrastructure as code.

### CODE REVIEW

File: docs/submissions.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ This is the place to add your own Cloud Resume API project!
 | Toyyib Muhammad-Jamiu | https://zu27je734ldpvy2kic6h57wqy40agiuu.lambda-url.us-east-1.on.aws/  | https://github.com/Abunuman/Abunuman-AWS-Cloud-Resume-API-Challenge |
 | George Wettlaufer | https://ry5dzf3ztf.execute-api.us-east-1.amazonaws.com/prod/cloud-resume | https://github.com/loggerboy9325/cloud-resume-api |
 | Meriem Terki | https://lpn6ekqu21.execute-api.us-east-1.amazonaws.com/prod/resume | https://github.com/MeriemTerki/aws-cloud-resume-api |
+|Anandha krishnan|https://jner4mrs02.execute-api.us-east-1.amazonaws.com/prod|https://github.com/Anandhakrishnan27/resumeApi
```
Comment (rishabkumar7) [inline docs/submissions.md:22]: Can you include the closing pipe | at the end of the record.

File: README.md
Comment (rishabkumar7) [inline README.md:1]: This is added automatically by the GitHub Action, can you revert this commit.

**Review by rishabkumar7 — CHANGES_REQUESTED** [review]

File: README.md
Comment (Anandhakrishnan27) [reply README.md:1]: i reverted the commit

File: docs/submissions.md (hunk 1 above)
Comment (Anandhakrishnan27) [reply docs/submissions.md]: i have included the pipe at the end

**Review by rishabkumar7 — APPROVED** [review]
LGTM, thank you.

**Comment by rishabkumar7** [comment]
Hi @Anandhakrishnan27 congrats on winning the AWS Credits, could you share your preferred social media so that I can share the details. Thank you.

**Comment by Anandhakrishnan27** [comment]
You can share it via my LinkedIn and my profile is
https://www.linkedin.com/in/anandhakrishnan27?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app

**Pull request merged**